torch/csrc/cuda/shared/nvtx.cpp

```diff
@@ -1,22 +1,206 @@
 #ifdef _WIN32
 #include <wchar.h> // _wgetenv for nvtx
 #endif
+
 #include <nvToolsExt.h>
 #include <torch/csrc/utils/pybind.h>
 
 namespace torch {
 namespace cuda {
 namespace shared {
 
+#ifndef USE_ROCM
+
+// registry idea from NVIDIA/nvtx-plugins  @jdekhtiar
+class DomainCategoryRegistry {
+ public:
+  DomainCategoryRegistry() {}
+
+  ~DomainCategoryRegistry() {
+    for (auto& domain : domains)
+      nvtxDomainDestroy(domain.second);
```

Comment:
Wrap even single line for/if body in curly braces
```suggestion
    for (auto& domain : domains) {
      nvtxDomainDestroy(domain.second);
    }
```

Reply:
Fixed in 30804cc